### DISCUSSION TITLE
Methodology section: rephrase "software is emitting" carbon

### DISCUSSION THREAD

**Discussion by dschien** [Green-Software-Foundation/sci#194] [SCI feedback]
GHG are emitted, among others, during generation of electricity from fossil fuel - software is not emitting the carbon. 

I suggest rephrasing to refer to the carbon footprint of software.

**Comment by atg-abhishek** [comment]
Agreed - @dschien please feel free to propose the change once all the existing PRs are merged in during the WG call today. Thanks!
